### DISCUSSION TITLE
headless-ui/angular

### DISCUSSION THREAD

**Discussion by michaelbuller** [tailwindlabs/headlessui#360] [General]
Are you all working on an Angular package?  It would be good to know if it is in your roadmap before we spend cycles on implementing this.

**Comment by adamwathan** [comment]
Hey! We aren't currently no and no concrete plans for it right now.

**Reply by michaelbuller** [reply]
Ok, Thank you!  We will get started then.

**Comment by caoimhebyrne** [comment]
As an angular user, it would be quite nice to have angular support :-)

**Reply by michaelbuller** [reply]
I have made our package and progress open source.  Ill post more on the state of things as we go.

**Reply by caoimhebyrne** [reply]
> I have made our package and progress open source. Ill post more on the state of things as we go.

Awesome, thanks! :)

**Reply by mattbodman** [reply]
> I have made our package and progress open source. Ill post more on the state of things as we go.

Where is the repo?

**Comment by lume-code** [comment]
Based on GitHub stars and other indicators of popularity, it appears that React and Vue were developed first. However, as someone who has experience with both React and Angular, I find that Angular is better suited for larger enterprise applications. Many companies in this space tend to use Angular as their preferred framework. While I personally use React for landing page websites, I would never use it for applications. I would go to the extent of creating all the Angular components necessary to support Tailwind, as I believe these components are more beneficial for applications. I hope that you will eventually consider implementing these components in Angular too.

**Comment by gayuru** [comment]
+1 for Angular Support ❤️

**Comment by abomadev** [comment]
For anyone looking for Angular support, I built a Tailwind UI to Angular converter. Check it out at [twuing.dev](https://twuing.dev/)

**Reply by onkarjit** [reply]
Does it support accessibility?

**Comment by alexisAvenel** [comment]
+1 for Angular Support please =D

**Comment by raknjarasoa** [comment]
IMO, we already have `angular cdk` to handle these primitives. cf example [here](https://material.angular.io/cdk/menu/examples)

**Comment by drony** [comment]
+1 Angular Support

**Comment by thomastrapanese** [comment]
hi, if you plan an angular development, I offer to contribute to the development of the angular version

**Comment by Til0r** [comment]
+1 for Angular Support 🚀

**Comment by SOG-web** [comment]
You can check out primeng.

Primeng is not totally headless but the customisation is very easy and you can tailwind to add your class to it

**Reply by SOG-web** [reply]
If you would like to get your hands dirty you can clone this repo and extend it

https://github.com/benjaminforras/ngx-headlessui-old

**Comment by XhstormR** [comment]
2025-12-08 23:46:39
